src/core/types/addons.js

```diff
@@ -20,15 +20,39 @@ export type AddonAuthorType = {|
   url: string,
 |};
 
-export type AddonType = {|
+type ThemeData = {|
+  accentcolor?: string,
+  author?: string,
+  category?: string,
+  description?: string,
+  detailURL?: string,
+  footer?: string,
+  footerURL?: string,
+  header?: string,
+  headerURL?: string,
+  iconURL?: string,
+  id?: number,
+  name?: string,
+  previewURL?: string,
+  textcolor?: string,
+  updateURL?: string,
+  version?: string,
+|};
+
+/*
+ * This is the external API representation of an add-on.
+ *
+ * This is a detailed API response. Not all API responses
```

Comment:
Linking to the API docs from here would be cool just so developers could understand better the varied details in responses. Maybe just http://addons-server.readthedocs.io/en/latest/topics/api/addons.html ?

Reply:
good idea